packages/terra-clinical-onset-picker/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+ChangeLog
+=========
+
+Unreleased
+----------
```

Comment:
You need to add a description of your changes here. It can just be 'Initial stable release' for this one.

Reply:
I've updated the changelog to include this.